annotations/src/main/kotlin/com/thinkslynk/fabric/annotations/registry/RegisterBlockItem.kt

```diff
@@ -3,7 +3,7 @@ package com.thinkslynk.fabric.annotations.registry
 @Retention(AnnotationRetention.SOURCE)
 @Target(AnnotationTarget.CLASS)
 annotation class RegisterBlockItem(
-        val namespace: String,
-        val name: String,
+        val namespace: String = "",
+        val name: String = "",
```

Comment:
Are these actually optional? And if so, how are you resolving them?

Reply:
These are optional, yes, and if empty they will be set to the same value as the block annotation.

Reply:
Sounds good, just double checking.